bugsnag-cli-wrapper.js

```diff
@@ -0,0 +1,151 @@
+            {
+                /**
+                 *
+                 * @param {BugsnagOptions} options - Common Key-value pairs of options for the CLI.
+                 * @param {string|undefined} options.bundle - The bundle identifier. Optional.
+                 * @param {string|undefined} options.codeBundleId - The code bundle identifier. Optional.
+                 * @param {string|undefined} options.sourceMap - The source map file for the project. Optional.
+                 * @param {string|undefined} options.versionName - The version name for the project. Optional.
+                 * @param {string|undefined} options.bundleVersion - The version of the bundle. Optional.
+                 * @param {string|undefined} options.plist - The plist file for iOS projects. Optional.
+                 * @param {string|undefined} options.scheme - The scheme for iOS builds. Optional.
+                 * @param {string|undefined} options.xcodeProject - The Xcode project file for iOS builds. Optional.
+                 * @param {string} target - The path to the file or directory to upload (e.g., a bundle file or folder).
+                 * @returns {Promise<string>} - Resolves with the command's output or rejects with an error message.
+                 */
+                iOS: (options = {}, target = '') =>
+                    BugsnagCLI.run('upload react-native-ios', options, target),
+                /**
+                 *
+                 * @param {BugsnagOptions} options - Common Key-value pairs of options for the CLI.
+                 * @param {string|undefined} options.bundle - The bundle identifier. Optional.
+                 * @param {string|undefined} options.codeBundleId - The code bundle identifier. Optional.
+                 * @param {string|undefined} options.sourceMap - The source map file for the project. Optional.
+                 * @param {string|undefined} options.versionName - The version name for the project. Optional.
+                 * @param {string|undefined} options.appManifest - The manifest file for the app. Optional.
+                 * @param {string|undefined} options.variant - The variant of the project (e.g., production, staging). Optional.
+                 * @param {string|undefined} options.versionCode - The version code for the project. Optional.
+                 * @param {string} target - The path to the file or directory to upload (e.g., a bundle file or folder).
+                 * @returns {Promise<string>}
+                 */
+                Android: (options = {}, target = '') =>
+                    BugsnagCLI.run('upload react-native-android', options, target),
+            }
+        ),
+        /**
+         *
+         * @param {BugsnagOptions} options - Common Key-value pairs of options for the CLI.
+         * @param {string|undefined} options.baseUrl - The base URL for the project. Optional.
```

Comment:
I'm fairly sure you can use this syntax for optional parameters

```suggestion
         * @param {?string} options.baseUrl - The base URL for the project. Optional.
```